platforms/android/service/engine/AndroidManifest.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.opencv.engine"
-    android:versionCode="218@ANDROID_PLATFORM_VERSION_CODE@"
-    android:versionName="2.18" >
+    android:versionCode="219@ANDROID_PLATFORM_VERSION_CODE@"
+    android:versionName="2.20" >
```

Comment:
Let's use 3.x and 300@ANDROID_PLATFORM_VERSION_CODE@ versions to differentiate it with 2.4.